test/e2e/dra/test-driver/app/kubeletplugin.go

```diff
@@ -378,9 +430,17 @@ func (ex *ExamplePlugin) nodeUnprepareResource(ctx context.Context, claimName st
 }
 
 func (ex *ExamplePlugin) NodeUnprepareResources(ctx context.Context, req *drapbv1alpha3.NodeUnprepareResourcesRequest) (*drapbv1alpha3.NodeUnprepareResourcesResponse, error) {
+	ex.blockUnprepareResourcesMutex.Lock()
+	defer ex.blockUnprepareResourcesMutex.Unlock()
```

Comment:
Is there a reason why this is blocking differently than NodePrepareResources?

In NodePrepareResources, it's the individual `nodePrepareResource` which blocks.

Reply:
done